distbelief/optim/downpour_sgd.py

```diff
@@ -81,10 +72,12 @@ def step(self, closure=None):
             self.accumulated_gradients.zero_()
 
         # internal sgd update
-
-        self.internal_optim.step(closure)
+        for group in self.param_groups:
```

Comment:
Can we call `super().step` or `internal_optim.step(closure)` here instead of reimplementing this?

Reply:
super() here is the `Optimizer` base class, which has no step object